{"uuid": "1251ab91-5100-4847-9e82-551e4505d9aa", "children": ["4509a2cb-60b1-4df6-a8d2-5bc2feead3d2"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=86.0.4240.193)\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/hongmeisun/Desktop/shmcode/web_lemon/conftest.py\", line 36, in login\n    '18124762064', 'ss123456')\n  File \"/Users/hongmeisun/Desktop/shmcode/web_lemon/pages/login_page.py\", line 45, in login_success\n    self.browser.find_element(*self.phone_locator).send_keys ( name )\n  File \"/Users/hongmeisun/Library/Python/3.6/lib/python/site-packages/selenium/webdriver/remote/webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"/Users/hongmeisun/Library/Python/3.6/lib/python/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/hongmeisun/Library/Python/3.6/lib/python/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1605078715699, "stop": 1605078718094}], "start": 1605078715699, "stop": 1605078718211}